app/components/SCForm.js

```diff
@@ -94,14 +102,53 @@ class SCForm extends Component {
   }
 
   onChange(value) {
-    this.setState({ value });
+    this.errCheck(value);
+  }
+
+  errCheck(value) {
+
+    const formInfo = this.props.navigation.state.params.form;
+    var field_key, type, max, min, field_label;
+    var newState = this.setState({
+      showErrors: !this.state.showErrors,
+      message: 'hard coded message'
+    });
+    // gets the value of what has been entered and expected type
+    for(var i = 0; i < formInfo.fields.length; i++) {
+      //input
+      field_key = value[formInfo.fields[i].field_key];
+      //expected type
+      type = formInfo.fields[i].type;
+      // max and min values. i.e. length of string or max # of occurences
+      max = formInfo.fields[i].constraints.maximum;
+      min = formInfo.fields[i].constraints.minimum;
+      // if the field_key is expected to be a number, convert it to a number.
+      // this is getting cumbersome... Find a better way to do this.
+      if (type === 'number') {
```

Comment:
Check this up above the min max value assignment. There are some fields that don't have a min and a max so you will only want to check for constraints based on what the type is.